apptools/apptools-windows-tests/apptools/create_package.py

```diff
@@ -71,5 +71,23 @@ def test_create_package_missing_icon_startUrl(self):
         comm.clear("org.xwalk.test")
         self.assertEquals(return_code, 0)
 
+    def test_create_package_canary(self):
+        comm.setUp()
+        os.chdir(comm.XwalkPath)
+        comm.clear("org.xwalk.test")
+        os.mkdir("org.xwalk.test")
+        os.chdir('org.xwalk.test')
+        cmd = comm.HOST_PREFIX + comm.PackTools + \
+            "crosswalk-pkg --platforms=windows --crosswalk=canary " + comm.ConstPath + "/../testapp/create_package_missing_icon_startUrl/"
```

Comment:
It also can use the testapp create_package_basic, and the testapp only have an html file is also can be used.